### PR TITLE
chore(backend-read-me): add running section on backend.

### PR DESCRIPTION
**Pull request by irumvanselme**

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -44,6 +44,15 @@ To develop this application locally, follow these steps:
 
    After running the above command, the `postinstall` script in the [package.json](package.json) will also run, and it will build and link the [api-specifications](/api-specifications/readme.md) dependency.
 
+## Running
+
+This backend is not designed for local development, you can't run it locally,
+
+>It is specifically designed for deployment on [AWS Lambda](https://aws.amazon.com/lambda/), utilizing [plumni](https://www.pulumi.com) for its construction. It is composed of cloud functions, which are not designed for local execution. This architecture choice emphasizes the backend's alignment with cloud-based environments, making it unsuitable for local development or API testing throught the REST Client apps such as postman.
```
Comment (rav3n11) [inline backend/README.md:51]: small typo on "through"

Comment (rav3n11) [inline backend/README.md:51]: small typo on "pulumi"

Comment (rav3n11) [reply backend/README.md:51]: If we are going to mention aws we should call the cloud functions by their native name on aws, Lambda functions...

**Review by rav3n11 — APPROVED** [review]
LGTM

File: backend/README.md [hunk 2]
```diff
@@ -44,6 +44,15 @@ To develop this application locally, follow these steps:
 
    After running the above command, the `postinstall` script in the [package.json](package.json) will also run, and it will build and link the [api-specifications](/api-specifications/readme.md) dependency.
 
+## Running
+
+This backend is not designed for local development, you can't run it locally,
+
+>It is specifically designed for deployment on [AWS Lambda](https://aws.amazon.com/lambda/), utilizing [pulumi](https://www.pulumi.com) for its construction. It is composed of aws, Lambda functions, which are not designed for local execution. This architecture choice emphasizes the backend's alignment with cloud-based environments, making it unsuitable for local development or API testing through the REST Client apps such as postman.
```
Comment (jacob-js) [inline backend/README.md:51]: Another detail to pay attention to is that, in this README we don't explain specific concepts. Therefore, the part `This architecture choice emphasizes the backend's alignment with cloud-based environments, making it unsuitable for local development or API testing through the REST Client apps such as postman.` isn't important.

**Note:** We focus on the conciseness and clarity of the information we are providing in this file.

File: backend/README.md [hunk 3]
```diff
@@ -44,6 +44,15 @@ To develop this application locally, follow these steps:
 
    After running the above command, the `postinstall` script in the [package.json](package.json) will also run, and it will build and link the [api-specifications](/api-specifications/readme.md) dependency.
 
+## Running
+
+This backend is not designed for local development, you can't run it locally,
```
Comment (jacob-js) [inline backend/README.md:49]: Let's make this:

`The backend project can't be run locally.`

**Review by jacob-js — CHANGES_REQUESTED** [review]

File: backend/README.md (hunk 3 above)
Comment (irumvanselme) [reply backend/README.md]: Fixed

File: backend/README.md (hunk 2 above)
Comment (irumvanselme) [reply backend/README.md]: Fixed

**Review by jacob-js — APPROVED** [review]

**Pull request merged**